sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Diagnostics/BlobCheckpointStoreInternal.Diagnostics.cs

```diff
@@ -106,10 +106,10 @@ static BlobCheckpointStoreInternal()
                                            string consumerGroup,
                                            string clientIdentifier,
                                            string sequenceNumber,
-                                           string replicationSegment,
+                                           string globalOffset,
```

Comment:
Technically a breaking change and not ideal - but replication segment is just a placeholder value in the current SDK. Alternatively could just tack global offset on the end and leave replication segment

Reply:
I think it's a reasonable change.   The type signature is the same.   It does break anyone using the name, but since the data was never actually populated, I think it's reasonable and low risk.